Chore/v1.0.0 beta.90 (#606)
Redocly/openapi-cli: docs/changelog.md

# OpenAPI CLI changelog

## 1.0.0-beta.90 (2022-03-24)

### Fixes

> - Update types used to lint .redocly.yaml according to new config file structure.

Reviewer: ```suggestion
- Updated types to support validation of the Redocly configuration file according to the new file structure.
```
